eth-types/src/evm_types/opcode_ids.rs

```diff
@@ -663,6 +663,162 @@ impl OpcodeId {
         }
     }
 
+    /// Returns the constant min & stack pointer of `OpcodeId`
+    pub const fn stack_pair(&self) -> (u32, u32) {
```

Comment:
what about rename this to something like "valid_stack_ptr_range_before_exec"?   self-explainable.

Reply:
`valid_stack_ptr_range` good name !

Reply:
updated in [3c9c55d](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/873/commits/3c9c55da2df49e8f6a236241b8d42fcfe6f4f465) thanks!